proposals/0000-structured-imports.rst

```diff
@@ -233,35 +205,39 @@ The new semantics are to be guarded by a language pragma, such as:
 
 Examples
 --------
-To clarify the above new import rules, suppose the module A has the following import/export structure::
 
-   module A
-     ( module ABC qualified
-     , module DEF qualified
-     )
-   where
-   import A.B.C as ABC (a,b,c)
-   import D.E.F as DEF (d,e,f)
+* Defining module::
 
-Then this table shows what names are brought into scope by the specified import statement:
+    {-# LANGUAGE StructuredImports #-}
 
-.. list-table:: Effects of import statements
-   :header-rows: 1
+    module Containers
+      ( module Map qualified          -- Export the set of names qualified with 'Map' and 'Set', qualified.
+      , module Set qualified          -- ..and the same for 'Set'.
+      , module Containers             -- And the 'Map' and 'Set' types, unqualified.
```

Comment:
This has to be `Map`, `Set`, listed explicitly, as they are not defined in `Containers`.

Reply:
Done, thanks!